pgjdbc/src/test/java/org/postgresql/replication/ReplicationSlotTest.java

```diff
@@ -80,9 +83,68 @@ public void testCreatePhysicalSlot() throws Exception {
 
     boolean result = isPhysicalSlotExists(slotName);
 
+    assertThat("Slot should exist", result, CoreMatchers.equalTo(true));
+
+    result = isSlotTemporary(slotName);
+
+    assertThat("Slot should not be temporary by default", result, CoreMatchers.equalTo(false));
+  }
+
+  @Test
+  public void testCreateTemporaryPhysicalSlot() throws Exception {
+    BaseConnection baseConnection = (BaseConnection) replConnection;
```

Comment:
The more idiomatic way of dealing with checking the version in the test is:
    assumeTrue(TestUtil.haveMinimumServerVersion(con, ServerVersion.v10_0));
as the first line